Examples/ReceiveDecryptAndValidate/MockAddressRegistry.cs

```diff
@@ -18,17 +18,17 @@ public Task<CommunicationPartyDetails> FindCommunicationPartyDetailsAsync(int he
             throw new NotImplementedException();
         }
 
-        public Task<CertificateDetails> GetCertificateDetailsForEncryptionAsync(int herId)
+        public Task<CertificateDetails> GetCertificateDetailsForEncryptionAsync(int herId, ICertificateValidator certificateValidator = null)
```

Comment:
Hvorfor ikke på constructor som foreslått?

Reply:
Mest for å teste og se bruk utover i stacken. Skal endre før jeg setter til "Ready for Review"

Reply:
Ønsker bare å bli mer kjent med messaging som en kodebase og ikke bare som et "konsept"